### ISSUE TITLE
Getting reports by issue id Error

### ISSUE THREAD

**Issue by bfulop** [kriskbx/gitlab-time-tracker#58]
Hello,

Thanks for this great utility, just exactly what I needed to track time with GitLab!

I just installed the repo (via npm, node v10.2.1) and created the configuration file. 

First, I was trying to add a list of issues to report on. Each time I add an issue `id`, I get this error:

```
$ gtt report 112
🔍  Resolving "112"... ✗
Error: Project not found or no access rights "112".
```

The project path is saved in the `config.yml` file. I can just run `gtt report` and it will start to process the issues. It does error out when trying to process the issues, but I think there are one or two issues that may have invalid `spend` value.

I also tried `$ got report "name/project" 112` and it gives the same error.

**Comment by kriskbx** [comment]
Thank you. :)
And thanks too for reporting this bug. I just released version 1.7.11 which should fix it. Please upgrade and try again. Feel free to reopen the issue if your problem persists.

**Comment by bfulop** [comment]
Thank you @kriskbx for the quick fix, it's working great now!
